shell/platform/fuchsia/dart_runner/dart_runner.cc

```diff
@@ -192,7 +222,7 @@ DartRunner::~DartRunner() {
 void DartRunner::StartComponent(
     fuchsia::sys::Package package,
     fuchsia::sys::StartupInfo startup_info,
-    ::fidl::InterfaceRequest<fuchsia::sys::ComponentController> controller) {
+    fidl::InterfaceRequest<fuchsia::sys::ComponentController> controller) {
```

Comment:
Ty for this, I've been trying to clean up this leading :: on ::fidl wherever I see them